test/release/examples/primers/chplvis/prog2.chpl

```diff
@@ -0,0 +1,44 @@
+// Example 2 of use of VisualDebug module and chplvis tool.
```

Comment:
You may want to take a look at the other primers in the directory.  They're a little more verbose and comment rich (though I think this file does get the point across).  I think it's fair to have this code live here, but the html should probably live elsewhere, like the website.